byway/src/main/webapp/interests.html

```diff
@@ -9,6 +9,10 @@
   <body>
     <div id="header">Some content</div>
     <p> Scroll through and select some interests</p>
-    <div id="interests"></div>
+    <div id="interests-section"></div>
+    <form id="interests-form">
+      <input type="hidden" id="interests" name="interests" />
+      <input type="submit" value="Next page" />
+    </form>
```

Comment:
What is the purpose of this empty form for the next button? You can just add `addEventListener('click', ...)` to a button rather than create a form for it, if you aren't going to use the form data for anything.

Reply:
I updated to make the form send the data and add another explicit link to the next page. I thought I could set the action to this form to link to the next page since it was working with an earlier implementation, but it does not do what I intend to do. Also, the action attribute isn't meant for that but rather to send the form data to that action page.

Fixed